hssqlppp/src/Database/HsSqlPpp/Parsing/ParserInternal.lhs

```diff
@@ -1206,10 +1206,15 @@ parse it
 >         pVol = matchAKeyword [("volatile", Volatile)
 >                              ,("stable", Stable)
 >                              ,("immutable", Immutable)]
->         readLang = keyword "language" *> matchAKeyword [("plpgsql", Plpgsql)
->                                                        ,("sql",Sql)]
+>         readLang = keyword "language" *> matchAKeyword [ ("plpgsql", Plpgsql)
+>                                                        , ("sql",Sql)
+>                                                        , ("python",Python)
+>                                                        ]
 >         parseBody :: ParseFlags -> Language -> ScalarExpr -> MySourcePos
 >                   -> Either String FnBody
+>         parseBody _ Python body _ = case body of
+>           StringLit _ value -> pure $ PythonFnBody value
+>           _ -> Left $ "Missing function code (Python). Expected $$function_code$$."
```

Comment:
This error means that the user used a value of the wrong type (not string literal) instead. This should probably never happen, but just in case we make a mistake I think it's better to provide something a bit more explanatory:

"Unexpected value for python user defined function body. Expecting a string literal."